.github/workflows/test-build.yml

```diff
@@ -41,6 +41,19 @@ jobs:
           exit 1
           fi
 
+      # Test TypeScript gRPC code snippets
+      - name: Compile TypeScript gRPC code snippets
+        run: npm install --prefix code_snippets/ts-grpc && npm run proto --prefix code_snippets/ts-grpc && npm run build --prefix code_snippets/ts-grpc
+
+      - name: Check if TypeScript gRPC code snippets build is successful
+        run: |
+          if [ -d "code_snippets/ts-grpc/dist" ]; then
+          echo "TypeScript gRPC code snippets compilation successful"
+          else
+          echo "TypeScript gRPC code snippets compilation failed"
+          exit 1
+          fi
+
```

Comment:
Follow-up idea: We could factor out testing compiling the sources into a GH action and then share it between the `pre-release.yml` and the `test-build.yml` workflow. An example would be https://github.com/restatedev/restate/blob/main/.github/actions/install-protoc/action.yml.